lib/perl/Genome/Model/Tools/Manta/Base.pm

```diff
@@ -0,0 +1,246 @@
+package Genome::Model::Tools::Manta::Base;
+
+use strict;
+use warnings;
+
+use Genome;
+use Carp qw(confess);
+
+class Genome::Model::Tools::Manta::Base {
+    is => 'Command',
+    is_abstract => 1,
+    attributes_have => [
+        tool_arg_name => {
+            is => 'String',
+            is_optional => 1,
+        },
+        tool_input_file => {
+            is => 'Boolean',
+            is_optional => 1,
+        },
+    ],
+    has_param => [
+        version => {
+            is => 'Version',
+            doc => 'The version of Manta to use.',
+            valid_values => ['0.29.6'],
+        },
+    ],
+    has_optional_output => [
+        output_files => {
+            is_many => 1,
+        },
+        input_files => {
+            is_many => 1,
+        },
+    ],
+};
+
+sub help_detail {
+    "Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads."
+}
+
+my %TOOL_VERSIONS = (
+    '0.29.6'     => '/gscmnt/gc13001/info/model_data/jwalker_scratch/src/manta-0.29.6.centos5_x86_64/bin',
```

Comment:
It is the same as using normal lsf queue.  For example,

`   bsub -R 'select[mem>16000] rusage[mem=16000]' -M 16000000 -Is -q docker -a "docker(registry.gsc.wustl.edu/sleong/manta:0.29.6)" /usr/local/pkg/manta-0.29.6.centos5_x86_64/bin/configManta.py --normalBam "/gscmnt/gc9022/info/build_merged_alignments/merged-alignment-blade13-4-5.gsc.wustl.edu-jwalker-25191-5e2355d1bb8c4ff78279ddb3947fc0d3/5e2355d1bb8c4ff78279ddb3947fc0d3.bam" --referenceFasta "/gscmnt/ams1102/info/model_data/2869585698/build106942997/all_sequences.fa" --tumorBam "/gscmnt/gc8002/info/build_merged_alignments/merged-alignment-blade13-2-11.gsc.wustl.edu-jwalker-27691-f1e64d1802b0493095d70b837a467f47/f1e64d1802b0493095d70b837a467f47.bam" --runDir "/tmp/"`